Source/WebCore/rendering/RenderFlexibleBox.h

```diff
@@ -92,6 +92,7 @@ class RenderFlexibleBox : public RenderBlock {
 
     bool shouldApplyMinBlockSizeAutoForChild(const RenderBox&) const;
 
+    bool isComputingBaseSizes() const { return m_isComputingBaseSizes; }
```

Comment:
Style nit: please add an empty line after the method.

Reply:
Fixed